app/view_models/admin/edit_auction_view_model.rb

```diff
@@ -14,15 +14,27 @@ def delivery_due_partial
   end
 
   def date_default(field)
-    dc_time(field).to_date
+    if field == 'paid'
```

Comment:
ah yeah we actually don't need these now -- added when I was trying to put payment fields into form, which was getting complicated bc we wanted to default to nil. Will remove for now ;)